packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+  //           v: 28,
+  //         }, {
+  //           r: "0x9294c656bdb6764fca46e431dc4b15c653e6347a41eb657d23145d93a1fa19d0",
+  //           s: "0x7afe0be470e9393dda32c356a9a262f7794a59f8e75e551bdb7634beb3a0a114",
+  //           v: 28,
+  //         }
+  //       ],
+  //       derivedKeyId: "0961c21c8a46c4992003a7b7af9449c15f772a269633ae3242f6ed146708a819",
+  //     }
+  //   },
+  //   signatures: {},
+  //   decryptions: [],
+  //   response: undefined,
+  //   logs: "",
+  // }
+
+  expect(res?.claims?.['foo']).toBeDefined();
+  expect(res?.claims?.['foo']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['foo'].signatures).toBeDefined();
+
+  res?.claims?.['foo'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+};
+
+const pkpSign = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.pkpSign({
+    toSign: alice.loveLetter,
+    pubKey: alice.authMethodOwnedPkp?.publicKey!,
+    sessionSigs: litActionSessionSigs!,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  // -- Expected output:
+  // {
+  //   r: "ab2cef959db920d56f001c3b05637ee49af4c4441f2867ea067c413594a4c87b",
+  //   s: "4bf11e17b4bb618aa6ed75cbf0406e6babfd953c5b201da697077c5fbf5b542e",
+  //   recid: 1,
+  //   signature: "0xab2cef959db920d56f001c3b05637ee49af4c4441f2867ea067c413594a4c87b4bf11e17b4bb618aa6ed75cbf0406e6babfd953c5b201da697077c5fbf5b542e1c",
+  //   publicKey: "04400AD53C2F8BA11EBC69F05D1076D5BEE4EAE668CD66BABADE2E0770F756FDEEFC2C1D20F9A698EA3FEC6E9C944FF9FAFC2DC339B8E9392AFB9CC8AE75C5E5EC",
+  //   dataSigned: "7D87C5EA75F7378BB701E404C50639161AF3EFF66293E9F375B5F17EB50476F4",
+  // }
+
+  expect(res?.r).toBeDefined();
+  expect(res?.s).toBeDefined();
+  expect(res?.dataSigned).toBeDefined();
+  expect(res?.publicKey).toBeDefined();
+};
+
+const executeJsJSONResponse = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+    console.log('hello world')
+
+    LitActions.setResponse({
+      response: JSON.stringify({hello: 'world'})
+    });
+
+  })();`,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
```

Comment:
Need this to be in a finally{} block so that if code throws on L490-502, we release the key